app/mattermost.js

```diff
@@ -69,7 +69,7 @@ export default class Mattermost {
         this.isConfigured = false;
         this.allowOtherServers = true;
 
-        Orientation.lockToPortrait();
+        Orientation.unlockAllOrientations();
```

Comment:
Do we need to explicitly unlock the orientation, or is that the default?

Reply:
needed or it will start as portrait